bin/build.js

```diff
@@ -2,24 +2,24 @@ const optimize = require('./optimize');
 const generatePathFile = require('./path-data');
 
 module.exports = function () {
-  return optimize(['*.svg'], './')
-    .then(function (result) {
-      return optimize(['icons/*.svg'], 'icons/', true);
+  return optimize('*.svg')
+    .then(function () {
```

Comment:
Can you use fat arrow syntax for conciseness?

Reply:
@jcfranco the icon team's automated workflow runs on a pretty ancient node version, once they update that machine I can clean a lot of this up.

Reply:
Actually they are on 8, which I think can handle this, will update :+1: